flink-runtime/src/test/java/org/apache/flink/runtime/io/network/api/writer/RecordWriterTest.java

```diff
@@ -311,7 +334,7 @@ public void testBroadcastEventBufferReferenceCounting() throws Exception {
 
 		ResultPartitionWriter partition =
 			new CollectingPartitionWriter(queues, new TestPooledBufferProvider(Integer.MAX_VALUE));
-		RecordWriter<?> writer = RecordWriterBuilder.createRecordWriter(partition);
+		RecordWriter<?> writer = createRecordWriter(partition);
```

Comment:
If this is a test fix, not a refactor, this change should be in separate commit. However I think this is going to be changed anyway, because of my pending  "Builder pattern" comment/issue.